contentcuration/contentcuration/templatetags/perseus_tags.py

```diff
@@ -1,15 +1,10 @@
-import re
+import json
 from django import template
 from django.template.defaultfilters import stringfilter
 
 register = template.Library()
 
 @register.filter
 @stringfilter
-def escape_newline(value):
-	return value.replace('\n', '\\n')
-
-@register.filter
-@stringfilter
 def escape_chars(value):
-	return value.replace('\n', '\\n').replace('\\"', '"')
\ No newline at end of file
+    return json.dumps(value)[1:-1]
```

Comment:
Maybe "easier" isn't the right word -- but more generalizable, as it would then work for non-string values, too (e.g. an int, or an object). And then perhaps the filter should be called "jsonify"?